config.py

```diff
@@ -80,56 +82,32 @@
-                  'cloak', 'shadow blade', 'diffusal blade', 'tango', 'dust of appearance', 'belt of strength',
-                  'hurricane pike', 'vladmir s offering', 'tranquil boots', 'javelin', 'meteor hammer', 'broadsword',
-                  'ultimate orb'}
-
-# Note: Get them from here
-# https://dota2.gamepedia.com/api.php?action=cargoquery&tables=heroes&fields=title&where=game+IS+NULL&limit=500&format=json
-HERO_NAME_RESPONSES = {'silencer', 'phantom assassin', 'clinkz', 'huskar', 'juggernaut', 'crystal maiden', 'pudge',
-                       'disruptor', 'queen of pain', 'wraith king', 'spectre', 'templar assassin', 'warlock',
-                       'earth spirit', 'viper', 'slark', 'weaver', 'alchemist', 'treant protector', 'axe', 'tidehunter',
-                       'invoker', 'kunkka', 'keeper of the light', 'undying', 'phoenix', 'terrorblade', 'doom',
-                       'broodmother', 'death prophet', 'earthshaker', 'mirana', 'storm spirit', 'bounty hunter',
-                       'clockwerk', 'lina', 'magnus', 'lifestealer', 'enigma', 'windranger', 'dark seer', 'drow ranger',
-                       'tiny', 'chaos knight', 'vengeful spirit', 'nyx assassin', 'ancient apparition', 'tusk',
-                       'ember spirit', 'io', 'outworld devourer', 'lion', 'underlord', 'lone druid', 'ursa', 'batrider',
-                       'riki', 'sven', 'ogre magi', 'beastmaster', 'anti mage', 'morphling', 'medusa', 'arc warden',
-                       'shadow demon', 'naga siren', 'slardar', 'bloodseeker', 'winter wyvern', 'leshrac', 'lycan',
-                       'omniknight', 'witch doctor', 'shadow shaman', 'sand king', 'necrophos', 'faceless void',
-                       'pangolier', 'grimstroke', 'dazzle', 'visage', 'spirit breaker', 'centaur warrunner',
-                       'monkey king', 'jakiro', 'dragon knight', 'abaddon', 'pugna', 'dark willow', 'night stalker',
-                       'luna', "nature s prophet", 'lich', 'bane', 'mars', 'phantom lancer', 'troll warlord', 'chen',
-                       'techies', 'skywrath mage', 'enchantress', 'razor', 'gyrocopter', 'tinker', 'zeus', 'meepo',
-                       'rubick', 'elder titan', 'brewmaster', 'venomancer', 'shadow fiend', 'puck', 'legion commander',
-                       'sniper', 'oracle', 'timbersaw', 'bristleback', 'snapfire', 'void spirit'}
+#Hero and item responses not hardcoded here
+# Drawbacks: Can't be tweaked
+#TODO incorperate an  better exclusion set and populate it with the difference of responses from main
+SPECIAL_CHARACTERS = ["'", "(", ")", ":", ",", "-", "."]
+
+with urllib.request.urlopen('https://dota2.gamepedia.com/api.php?' +
+                            'action=cargoquery&tables=heroes&fields=title&where=game'+
+                            '+IS+NULL&limit=500&format=json') as url:
+    SELECTED_HERO_RESPONSES = {}
+    HERO_NAME_RESPONSES = set()
+    for i in json.loads(url.read().decode())['cargoquery']:
```

Comment:
`i` does not say much about what the variable is, would be better to name it in a more verbose way for better readability and future understanding of the code.

Reply:
I have changed around the variable names to hopefully be more verbose and to increase readability.  Please let me know if there is any confusion with the names and I can find something that works better.